augur/application/cli/backend.py

```diff
@@ -93,7 +94,19 @@ def start(disable_collection, development, port):
         cpu_worker_process = subprocess.Popen(cpu_worker.split(" "))
         time.sleep(5)
 
-        start_task.si().apply_async()
+        with DatabaseSession(logger) as session:
```

Comment:
This could possibly be the source of the connection bloating I am seeing at runtime, including high volumes of rollback transactions.